edgelet/iotedge/src/system.rs

```diff
@@ -0,0 +1,65 @@
+// Copyright (c) Microsoft. All rights reserved.
+
+use std::ffi::OsStr;
+
+use crate::error::{Error, ErrorKind};
+use aziotctl_common::{
+    get_status, get_system_logs as logs, restart, set_log_level as log_level, ServiceDefinition,
+    SERVICE_DEFINITIONS as IS_SERVICES,
+};
+
+lazy_static! {
+    static ref IOTEDGED: ServiceDefinition = {
+        let sockets: &'static [&'static str] = option_env!("IOTEDGE_HOST").map_or(&[], |_host| {
```

Comment:
@arsing I wasn't sure if this is right. I know CentOS doesn't use the sockets, but is CentOS the only distro that doesn't set "IOTEDGE_HOST"?

Reply:
CentOS is the only distro that *does* set `IOTEDGE_HOST`